docs/docs/quickstart.mdx

```diff
@@ -0,0 +1,52 @@
+---
+sidebar_position: 3
+title: Quickstart
+---
+
+# Quickstart
+
+Follow these steps for a quick start:
+
+1. Install Chainloop. This quick snippet will get you through the process:
+    ```bash
+    curl -sfL https://docs.chainloop.dev/install.sh | bash -s
+    ```
+    Refer to [these instructions](/getting-started/installation) for more installation options.
+
+2. Authenticate to the Control Plane via OIDC by running:
+    ```bash
+    chainloop auth login
+    ```
+    This will create a personal token that you will use in operations not related to attestations.
+
+3. Create a new Chainloop workflow.
+    ```bash
+    chainloop wf create --name mywf --project myproject
+    ```
+    Chainloop workflows represent any CI or process you might want to attest.
+    Check [this doc](/getting-started/workflow-definition)
+    for a complete explanation of Workflows and Contracts.
+    You might also want to check our [contract reference](/reference/operator/contract).
+
+4. Create a restricted token for further operations:
+    ```bash
+    export CHAINLOOP_TOKEN=$(chainloop org api-token create --name test-api-token -o json | jq -r ".[].jwt")
```

Comment:
some people might not have jq, I am wondering if we should add a `raw` or `token` output to the command, wdyt?

Reply:
Yes, I was thinking that today. Maybe we could output everything to stderr, and the token to stdout.